PizzaAPI/PizzaAPI/Models/Comments.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace PizzaAPI.Models
+{
+    public class Comments
```

Comment:
Comments to liczba mnoga więc sugeruje kolekcję jakiegoś typu. Tutaj powinien być `Comment`, a prop w nim np. `Content`